New list snippets (#188)
30-seconds/30-seconds-of-python: includes_any.md

---
title: includes_any
tags: list,intermediate
---

> Returns `True` if any the elements in `values` are included in `lst`, `False` otherwise.

Reviewer: ```suggestion
Returns `True` if any element in `values` is included in `lst`, `False` otherwise.
```
